pkgs/by-name/li/linkding/package.nix

```diff
@@ -0,0 +1,134 @@
+{
+  fetchFromGitHub,
+  fetchNpmDeps,
+  lib,
+  nix-update-script,
+  nixosTests,
+  nodejs,
+  npmHooks,
+  pkgs,
+  playwright-driver,
+  python3Packages,
+  stdenvNoCC,
+  uwsgi,
+}:
+let
+  pname = "linkding";
+  version = "1.36.0";
+
+  src = fetchFromGitHub {
+    owner = "sissbruecker";
+    repo = "linkding";
+    tag = "v${version}";
+    hash = "sha256-AePxd7uc1DvMwzhWluYIzXnFDlH/kiWNjBMzj/p68Mk=";
+  };
+  npmDeps = fetchNpmDeps {
+    inherit src;
+    name = "${pname}-npm-deps";
+    hash = "sha256-BA7PurUcB5glgaBIpuZLPRSg3GDdi4CLniUjoa/32yc=";
+  };
+
+in
+python3Packages.buildPythonApplication {
+  inherit
+    npmDeps
+    pname
+    src
+    version
+    ;
+
+  build-system = [
+    python3Packages.setuptools
+  ];
+
+  dependencies = [
+    python3Packages.asgiref
+    python3Packages.beautifulsoup4
+    python3Packages.bleach
+    python3Packages.bleach-allowlist
+    python3Packages.certifi
+    python3Packages.cffi
+    python3Packages.charset-normalizer
+    python3Packages.click
+    python3Packages.confusable-homoglyphs
+    python3Packages.cryptography
+    python3Packages.django
+    # python3Packages.django-registration # Not in nixpkgs
```

Comment:
I don't know whether this is actually needed for anything.

Reply:
Yes, I think it is, if you enable ALLOW_REGISTRATION

https://github.com/sissbruecker/linkding/blob/d2e8a95e3cccd58d7d08be9692a5e6ed5945b2af/siteroot/urls.py#L82-L83